runtime/nls/cfre/cfrerr_ca.nls

```diff
@@ -67,6 +67,7 @@ J9NLS_CFR_ERROR_TEMPLATE_METHOD.sample_input_8=5
 J9NLS_CFR_ERROR_TEMPLATE_METHOD.explanation=NOTAG
 J9NLS_CFR_ERROR_TEMPLATE_METHOD.system_action=
 J9NLS_CFR_ERROR_TEMPLATE_METHOD.user_response=
+
```

Comment:
This PR is undoing a number of changes we just made. Merging this PR will undo them all, creating a mismatch with the latest cfrerr.nls file. I suppose it doesn't matter that much, as we'd pick up the changes again in the next translation cycle, so I could overlook it.